src/m1_facilitator/repositories/AnchorRepository.ts

```diff
@@ -26,11 +26,11 @@ import assert = require('assert');
 /* eslint-disable class-methods-use-this */
 
 /**
- * An interface, that represents a row from an anchor table.
+ * An interface, that represents an anchor database model.
  *
  * See: http://docs.sequelizejs.com/manual/typescript.html#usage
  */
-class AnchorModel extends Model {
+class AnchorDatabaseModel extends Model {
```

Comment:
Let us keep it consistent with what we are following till now 
In case we want to change it to `AnchorDatabaseModel`, we can discuss and change it for all the files.

Reply:
I am not sure how it's going to affect the sprint and also the other parts. It's really confusing to name this as `AnchorModel` and also have `models/Anchor` which is again an anchor model. The proper naming here is AnchorDatabaseModel (that's what it is actually I believe). I will change the name as has been asked, however, I also want us to discuss if it's good naming instead of delaying it for future cases. Thx.

Reply:
We already had discussion on the naming when we were implementing M0-facilitator.
This point was already raised by the team then, but it was decided to go with the naming convention.
Lets go with the name `AnchorModel` rather than changing it for all the implemented models.

Reply:
Ok